services/catarse/spec/models/subscription_payment_spec.rb

```diff
@@ -3,5 +3,193 @@
 require 'rails_helper'
 
 RSpec.describe SubscriptionPayment, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe '#refund' do
+    let(:gateway_id) { '123' }
+    let(:subscription_payment) { described_class.new(gateway_general_data: { gateway_id: gateway_id }) }
+    let(:transaction) { double(status: 'paid', refund: true, payment_method: 'boleto') } # rubocop:disable RSpec/VerifiedDoubles
```

Comment:
**Sugestão**
Deixar aqui como cartão de crédito por padrão, e onde for necessário, mudar o meio de pagamento para outro.